release-notes/opensearch-performance-analyzer.release-notes-2.0.0.0-rc1.md

```diff
@@ -26,5 +25,6 @@ Compatible with OpenSearch 2.0.0-rc1
 * Update directory names ([#166](https://github.com/opensearch-project/performance-analyzer-rca/pull/166))
 * Update PA directories from plugins to root ([#168](https://github.com/opensearch-project/performance-analyzer-rca/pull/168))
 
-### Deprecation
+### Documentation
+* Updated issue templates from .github. ([#177](https://github.com/opensearch-project/performance-analyzer/pull/177))
 * Removing metrics which are not required now as were removed in OS 2.0 ([#159](https://github.com/opensearch-project/performance-analyzer-rca/pull/159))
```

Comment:
This commit is more of a ```Maintenance``` than a documentation change. What do you think?

Reply:
I mostly referred this from here - https://github.com/opensearch-project/opensearch-build/pull/2039 and saw mostly adding this under "Documentation" which also makes sense considering the changes done in that commit.